src/frontends/tensorflow/src/utils.cpp

```diff
@@ -4,10 +4,14 @@
 
 #include "utils.hpp"
 
+#include "openvino/opsets/opset10.hpp"
 #include "openvino/opsets/opset8.hpp"
 #include "openvino_conversions.hpp"
 
+using namespace ov;
+using namespace ov::opset10;
```

Comment:
it's best to avoid multiple such `usings` in the same file, as this can lead to inconsistencies between ov::opset10 and ov::opset8

Reply:
Agree, let me fix it in the next PR.